geonode/base/templates/base/_resourcebase_snippet.html

```diff
@@ -2,7 +2,7 @@
 {% verbatim %}
 <div class="row">
   <div ng-if="results.length == 0">
-    <div><h3>No {{ dataValue || 'content' }} created yet.</h3></div>
+    <div><h3>No content created yet.</h3></div>
```

Comment:
This was saying things like "No date created yet" due to dataValue storing the filter type, which was incorrect. A generic "No content created yet." should suffice.